CHANGELOG.rst

```diff
@@ -4,6 +4,231 @@
+    events on a vulnerability.
+
+    Add ``cve:desc``, ``cve:url``, and ``cve:references`` secondary properties
+    to record information about the CVE associated with a vulnerability.
+
+    Add ```nist:nvd:source`` to record the name of the organization which
+    reported the vulnerability in the NVD.
+
+    Add ``nist:nvd:published`` and ``nist:nvd:modified`` to record when the
+    vulnerability was first published, and later modified, in the NVD.
+
+    Add ``cisa:kev:name``, ``cisa:kev:desc``, ``cisa:kev:action``,
+    ``cisa:kev:vendor``, ``cisa:kev:product``, ``cisa:kev:added``,
+    ``cisa:kev:duedate`` properties to record information about the CISA KEV
+    database for the vulnerability.
+
+- Annotate the following light edges.
+  (`#2900 <https://github.com/vertexproject/synapse/pull/2900>`_)
+
+  ``seen``
+    When used with ``meta:source`` nodes, the edge indicates the target
+    node was observed by the source node.
+
+  ``uses``
+    When used with ``ou:campaign`` and ``ou:technique`` nodes, the edge
+    indicates the campaign used a given technique.
+
+    When used with ``ou:org`` and ``ou:technique`` nodes, the edge
+    indicates the organization used a given technique.
+
+    When used with ``risk:attack`` and ``ou:technique`` nodes, the edge
+    indicates the attack used a given technique.
+
+    When used with ``risk:threat`` and ``ou:technique`` nodes, the edge
+    indicates the threat cluster uses the technique.
+
+    When used with ``risk:tool:software`` and ``ou:technique`` nodes, the edge
+    indicates the tool uses the technique.
+
+    When used with ``risk:attack`` and ``risk:vuln`` nodes, the edge
+    indicates the attack used the vulnerability.
+
+    When used with ``risk:threat`` and ``risk:vuln`` nodes, the edge
+    indicates the threat cluster uses the vulnerability.
+
+    When used with ``risk:tool:software`` and ``risk:vuln`` nodes, the edge
+    indicates the tool used the vulnerability.
+
+    When used with ``risk:attack`` and ``ou:industry`` nodes, the edge
+    indicates the attack targeted the industry.
+
+    When used with ``risk:threat`` and ``ou:industry`` nodes, the edge
+    indicates the threat cluster targets the industry.
+
+    When used with ``risk:tool:software``, the edge indicates the target
+    node is used by the tool.
+
+  ``stole``
+    When used with a ``risk:compromise`` node, the edge indicates the target
+    node was stolen or copied as a result of the compromise.
+
+- Add ``$lib.gen.vulnByCve()`` to help generate ``risk:vuln`` nodes for CVEs.
+  (`#2903 <https://github.com/vertexproject/synapse/pull/2903>`_)
+- Add a unary negation operator to Storm expression syntax.
+  (`#2886 <https://github.com/vertexproject/synapse/pull/2886>`_)
+- Add ``$lib.crypto.hmac.digest()`` to compute RFC2104 digests in Storm.
+  (`#2902 <https://github.com/vertexproject/synapse/pull/2902>`_)
+- Update the Storm ``storm:http:resp`` ``.json()`` method to add optional
```

Comment:
this entry has a leading period ".json()" vs the next one which is just "decode()", probably remove the period